README.md

```diff
@@ -1,60 +1,95 @@
-right statistics, we can reliably identify small differences and quantify our
-confidence in them.
+## Example
 
-## Quick Start
+Let's test two approaches for adding elements to a page. First ceate two HTML
+files:
 
-1. Install tachometer from NPM.
+`inner.html`
 
-   ```sh
-   $ npm i tachometer
-   ```
+```html
+<script type="module">
+  import * as bench from '/bench.js';
+  bench.start();
+  for (let i = 0; i < 100; i++) {
+    document.body.innerHTML += '<button></button>';
+  }
+  bench.stop();
+</script>
+```
 
-2. Create a simple `forloop.html` micro benchmark that times a `for` loop.
-   tachometer benchmarks are HTML files that import and call `bench.start()` and
-   `bench.stop()`. Note that when you are measuring [first contentful
-   paint](#first-contentful-paint-fcp), you don't need to call these functions.
-
-   ```html
-   <html>
-     <body>
-       <script type="module">
-         import * as bench from '/bench.js';
-         bench.start();
-         for (let i = 0; i < 1000; i++) {}
-         bench.stop();
-       </script>
-     </body>
-   </html>
-   ```
+`append.html`
 
-3. Launch tachometer, which will launch Chrome and execute the benchmark 50
-   times.
+```html
+<script type="module">
+  import * as bench from '/bench.js';
+  bench.start();
+  for (let i = 0; i < 100; i++) {
+    document.body.append(document.createElement('button'));
+  }
+  bench.stop();
+</script>
+```
 
-   ```sh
-   $ tach forloop.html
-   ```
+Now run tachometer:
 
-   Along with some other information, tachometer will show you a range of
-   plausible values for how long this benchmark takes to run (more precisely, a
-   _95% confidence interval_, which is explained [below]()).
+```sh
+npx tachometer append.html inner.html
+```
 
-   <img src="./images/screen1.png">
+Tachometer opens Chrome and loads each HTML file, measuring the time between
+`bench.start()` and `bench.stop()`. It round-robins between the two files,
+running each at least 50 times.
+
+```
+[==============================================------------] 79/100 chrome append.html
+```
+
+After a few seconds, the results are ready:
+
+```
+┌─────────────┬─────────────────┬─────────────────┬─────────────────┐
+│ Benchmark   │        Avg time │   vs inner.html │  vs append.html │
+├─────────────┼─────────────────┼─────────────────┼─────────────────┤
+│ inner.html  │ 7.23ms - 8.54ms │                 │          slower │
+│             │                 │        -        │    851% - 1091% │
+│             │                 │                 │ 6.49ms - 7.80ms │
+├─────────────┼─────────────────┼─────────────────┼─────────────────┤
+│ append.html │ 0.68ms - 0.79ms │          faster │                 │
+│             │                 │       90% - 92% │        -        │
+│             │                 │ 6.49ms - 7.80ms │                 │
+└─────────────┴─────────────────┴─────────────────┴─────────────────┘
```

Comment:
It'd be great to also show an example where the confidence intervals overlap, so you can mention that either might have a faster true mean. I know that gets into the interpretation section, but it's kind of the key feature and you even mention "tiny differences" a few times, so it'd be good to show that.

Reply:
Yeah, something with a much smaller difference would be nice here I agree. I'm trying to think of an example that's equally simple, but struggling at the moment. I'll think of something later and do a followup -- any ideas?

We could do something like the for loop example that's in the "interpreting results" section, though it's pretty contrived. It's nice because it shows faster, slower, and unsure though, as you say. That requires 3 benchmarks, though, so it's a little more complex setup to show.

Reply:
There are some DOM APIs that are pretty close, but definitely different. Maybe appendChild vs insertBefore